pkg/cmd/generate-acceptance-tests/cli_test_template.go

```diff
@@ -31,8 +44,8 @@ var cmdBase = []string{
 	"/bin/bash",
 	"-c",
 }
-
-func TestDockerCLI(t *testing.T) {
+{{range $testSuffix, $testFilePath := .TestCases}}
+func TestDockerCLI_{{$testSuffix}}(t *testing.T) {
```

Comment:
It is generated code but the code should still be as readable as possible -- can you refactor these generated functions? It looks like you can just pass in the test suffix and test file path as arguments to some helper function which contains all the helper logic.

Reply:
Do you mean something like the following?
```
{{range $testSuffix, $testFilePath := .TestCases}}
func TestDockerCLI_{{$testSuffix}}(t *testing.T) {
    s := log.Scope(t)
    defer s.Close(t)
    runTestDockerCLI(t, testSuffix, testFilePath)
}
{{end}}
```

Reply:
Yes, that would work.